README.md

```diff
@@ -0,0 +1,62 @@
+
+Not only does the mod allow players to talk to each other in-game, but it also adds various communication-centric content to the game. But that will all be configurable.
+
+## Installation
+1. Download the latest version of the mod from the [mod page](https://mods.vintagestory.at/rpvoicechat).
+2. Place the downloaded file in the `Mods` folder of your Vintage Story installation.
+3. Start the game and enjoy!
+(4.) Optional: Double check your keybinds in the settings menu. All keybinds start with `RPVC` and are located in the hud and interface category.
+ 
+## Usage
+### Talking
+To talk to other players you will need to set up your audio input threshold or turn on push to talk (PTT) in the config menu.
+The default keybind for the config menu is `P` and the default keybind for PTT is `CAPS LOCK`. (Note as of current game version these keybinds cannot be applied to mouse buttons)
+
+### Different voice levels
+There are currently 3 different voice levels in the mod. These are whisper, normal and shout. 
+The default keybind to change between these is `Shift + Tab`. This can be changed in the settings menu.
+
+### Configuring audio
+You can configure your microphone volume and the volume of everyone else in your config menu.
+
+## <a name="configuration-users"></a>Configuration (For users)
+The mod is highly configurable and can be configured more deeply in the modconfig file. You'll find this file in the `ModConfig` folder in the same directory as your `Mods` folder. The file is called `rpvoicechat.json`.
+The settings relevant to users are:
+- `PushToTalkEnabled` - Whether push to talk is enabled. The default value is `false`.
+- `IsLoopbackEnabled` - The setting that defines whether you should be able to hear what you're transmitting. The default value is `false`.
+- `IsHUDShown` - Whether or not the HUD should be shown. The default value is `true`.
+- `IsMuted` - Whether or not you are muted. The default value is `false`.
+- `InputThreshold` - The current setting of your audio input threshold in percent (0-100). The default value is `20`.
+- `MaxInputThreshold` - This configures how finely tuned the input threshold should be. The smaller number the more sensitive the input will be. The default value is `0.24`.
+ 
+## <a name="configuration-server"></a>Configuration (For server owners)
+The mod is highly configurable and can be configured more deeply in the modconfig file. You'll find this file in the `ModConfig` folder in the same directory as your `Mods` folder. The file is called `rpvoicechat.json`.
+The settings relevant to server owners are:
+- `ManualPortForwarding` - Setting that determines whether the server should attempt to do UPnP itself or if it will use the following settings to set up a server itself. The default value is `false`.
```

Comment:
This setting is for both client and server as UDP clients have to open ports too. It is also worth mentioning that setting this to true means that you take full responsibility for opening ports and if they are not open mod will work incorrectly